components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -498,20 +571,7 @@ func (p *pg) insertOrReusePolicyMemberWithQuerier(ctx context.Context, policyID
 	return err
 }
 
-func (p *pg) getPolicyMembers(ctx context.Context, id string) ([]v2.Member, error) {
-	return p.getPolicyMembersWithQuerier(ctx, id, p.db)
-}
```

Comment:
The only one using this helper is now also transactional so we always need `WithQuerier`.